readthedocs/projects/tasks.py

```diff
@@ -297,19 +313,10 @@ def setup_vcs(self):
         self.setup_env.update_build(state=BUILD_STATE_CLONING)
```

Comment:
Also a problem here. The build won't update with this status since it's `finalize=False`.